### ISSUE TITLE
RNNT loss is constantly zero

### ISSUE THREAD

**Issue by Rajratnpranesh** [theblackcat102/edgedict#36]
I am trying to train the model but the loss is constant = '0'. I found that the loss output from the PyTorch binding of WarpRNNT is a zero tensor. I am getting an error of 'src/binding.cpp:151: unsupported data type'. Is this is the reason why WarpRNNT is not able to calculate loss (I charged the datatype from float16 to 32 as sum() operation was not possible with float16). Any pointer is appreciated.

ty

**Comment by theblackcat102** [comment]
Are you using half precision training? It's hard to debug based on so few information provided. It would be helpful if you can provide your training hyperparameters, environment versions before any conclusion can be made
